docs/third-party-developers/extensibility/data-store/collections.md

```diff
@@ -122,6 +126,86 @@ or
     -   _query_ `Object`: The query arguments for the collection, eg. `{ order: 'ASC', sortBy: Price }`.
     -   _ids_ `Array`: If the collection route has placeholders for ids you provide the values for those placeholders in this array (in order).
 
+### getFromState
+
+This selector will return the state from the collections store.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `object`: The state from the collections storew ith the following properties:
+   	- _namespace_ `string`: The route namespace for the collection, eg. `/wc/blocks`.
+    - _resourceName_ `string`: The resource name for the collection, eg. `products/attributes`.
+    - _query_ `object`: The query arguments for the collection, eg. `{ order: 'ASC', sortBy: Price }`.
+    - _ids_ `array`: If the collection route has placeholders for ids you provide the values for those placeholders in this array (in order).
+   	- _type_ `string`: type of the collections ie `items`.
```

Comment:
It appears that some parameters are missing, see https://github.com/woocommerce/woocommerce-blocks/blob/3fb2dfb941b8667f29b3689cbd16605d2079079e/assets/js/data/collections/selectors.js#L12-L20